tools/tasks/render_workflow.yaml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Set fact for colaltion path
+  ansible.builtin.set_fact:
+    collection_dir: "{{ collection_path + name }}"
+
+- name: Change to the collection directory
+  ansible.builtin.command:
+    cmd: "cd {{ collection_dir }}"
+    chdir: "{{ collection_dir }}"
```

Comment:
This won't work because `cd` is a shell builtin, not a command. This task wouldn't do anything anyways, though, since any directory change would only affect this task.